chineduogada/tst-fr
# chore(mock-data):employee mock data

#### What does this PR do?
provide employee mock data
#### Description of Task to be completed?
* provide mock data for employee registration
* provide mock for basic employee data
* provide mock for fetching a list of employees
#### How should this be manually tested?
* clone the repository
* cd into the project directory
* run `npm install`
* run `npm test`
* check the `src/mock/` directory to find mock files
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
#### Questions:
